iphone/Classes/TiAppiOSProxy.m

```diff
@@ -223,6 +223,10 @@ - (void)didTakeScreenshot:(NSNotification *)info
 
 - (void)didReceiveApplicationShortcutNotification:(NSNotification *)info
 {
+  if (!info.userInfo) {
```

Comment:
I think it is not required to check. The [place](https://github.com/appcelerator/titanium_mobile/blob/2c2d7008f837da6b19c106414441cf7fe5ea9055/iphone/TitaniumKit/TitaniumKit/Sources/API/TiApp.m#L1262)   from where notification is fired, userInfo is always set and is required. I think it is intentional to not check for info.userInfo to nil. It should never nil. In case it is nil it means there is issue in the Titanium SDK and it need to be fixed.